README.md

```diff
@@ -317,6 +320,27 @@ Return possible geocode(s) for each given address.
     DefaultTargetCoordinate[] geocodes = Arrays.stream(manyGeocodes)
                     .map(GeocodingResponse::getRepresentativeGeocodeOfRequest).toArray(DefaultTargetCoordinate[]::new);
 
+## Poi Reachability service
+
+Return reachable POIs from a list of sources
+
+    TravelOptions options = new TravelOptions();
+    options.setMaxEdgeWeight(900);
+    options.setEdgeWeightType(EdgeWeightType.TIME);
+    options.addSource(new DefaultSourceCoordinate("id1", 40.608155, -73.976636));
+    options.setOsmTypes(Collections.singleton(new PoiType("amenity", "restaurant")));
```

Comment:
can you leave a comment anywhere that this refers to Open Street Map poi types (with link to the different types)

maybe not here but maybe in the code somewhere?

Reply:
with a link to osm wiki?

Reply:
yeah, maybe? otherwiese people might not be able to define appropriate "key" "value" pairs to use it

Reply:
I added a link :)